internal/oci/runtime_vm.go

```diff
@@ -537,7 +537,7 @@ func (r *runtimeVM) DeleteContainer(c *Container) error {
 		return err
 	}
 
-	if _, err := r.task.Shutdown(r.ctx, &task.ShutdownRequest{ID: c.ID()}); err != nil {
+	if _, err := r.task.Shutdown(r.ctx, &task.ShutdownRequest{ID: c.ID()}); err != nil && !errors.Is(err, ttrpc.ErrClosed) {
```

Comment:
errors.Is() is not supported by go 1.10 it seems., which 1.17 technically supports. it's possible errors.Unwrap would do this for us?

Reply:
Hmm. Looking at blog posts*, seems that Unwrap was added as part of 1.13. I'll figure out the best ancient way to deal with this and update the PR soon.

*: https://blog.golang.org/go1.13-errors

Reply:
ah, here we're using  'github.com/pkg/errors` which is different than the gostdlib `errors` package https://github.com/pkg/errors

Reply:
Just for the record, I've privately spoken with @haircommander as the best way to proceed here is opening a new PR with the change.

Considering I'll open a new PR manually, I'd rather gather all the backports I have to do in a single PR per release.
As the changes are needed, I'll take some time to (re)test that everything still works as expected and (re)submit the PRs next week.

Thanks all for the reviews!